src/modules/rewards/components/RewardsActivitiesListItem.tsx

```diff
@@ -44,48 +45,69 @@ const RewardsActivitiesListItem: FC<RewardActivitiesListItemProps> = ({ userId,
   const [errorMessage, setErrorMessage] = useState('');
 
   const RewardsActivityIcon = (type: ActvityType) => {
-
-    if (type === "follow_push_on_discord") {
-      return <Discord width={48} height={48} />
+    if (type === 'follow_push_on_discord') {
+      return <Discord width={48} height={48} />;
     }
 
     if (type === 'follow_push_on_twitter') {
-      return <Twitter width={48} height={48} />
+      return <Twitter width={48} height={48} />;
     }
 
-    // return <PushDefaultRewards />
-    return <RewardsActivity />
-    // return <Discord />
-  }
+    return <RewardsActivity />;
```

Comment:
We already discussed that this was supposed to be a separate component and needs to go to a separate file, then why declare it here.
It will be redeclared multiple times for every item in the list like this.